.github/renovate-config.json5

```diff
@@ -2,8 +2,8 @@
   $schema: "https://docs.renovatebot.com/renovate-schema.json",
   onboarding: false,
   requireConfig: "ignored",
-  platform: "github",
-  repositories: ["jenkinsci/helm-charts"],
+//  platform: "github",
+//  repositories: ["jenkinsci/helm-charts"],
```

Comment:
```suggestion
  platform: "github",
  repositories: ["jenkinsci/helm-charts"],
```

Reply:
Good catch! I always have to disable this to test locally 😅